src/Application/Migrations/24_function_ticket_reset.sql

```diff
@@ -0,0 +1,75 @@
+BEGIN;
+
+CREATE OR REPLACE FUNCTION ticket_reset(param_ticket_id bigint)
```

Comment:
Mh, for a future extension of this function, it would be nice to limit the types of tickets to be reset.
When you have a recording ticket, you might not want to reset that to "scheduled" once it moved beyond the "recorded" state.
Maybe as a optional second parameter "enum_ticket_type[] param_ticket_types"

Reply:
Ok, there parameter is there now, but neither used in the model nor in the stored procedure either.
That's kind of pointless. Maybe add it as `-- TODO: ` comment for later, or implement the filter.